frontend/src/reducer.js

```diff
@@ -1,9 +1,26 @@
 /* eslint-disable no-fallthrough */
+
+import newYork from "./images/domestic/new-york.svg";
+import southDakota from "./images/domestic/south-dakota.svg";
+import lasVegas from "./images/domestic/las-vegas-nevada.svg";
+import losAngeles from "./images/domestic/los-angeles-california.svg";
+import washington from "./images/domestic/washington-dc.svg";
+import giza from "./images/international/giza-egypt.svg";
+import london from "./images/international/london-england.svg";
+import rio from "./images/international/rio-de-Janeiro-brazil.svg";
+
 const initialState = {
   lastGameComplete: {},
   currentGame: 1,
   games: {
     1: {
+      flightNumber: "QF3421",
+      journey: {
+        originName: "Las Vegas",
+        destinationName: "Los Angeles",
```

Comment:
I think you should keep the originName and originImage, and destinationName and destinationImage together, just a suggestion